uwblueprint/childrens-aid-society
# Get concerns given intake

## Notion ticket link

[Get concerns given intake id](https://www.notion.so/uwblueprintexecs/Get-concerns-given-intake-id-e8758394bc3f4c048c8593a785130e6f)



## Implementation description
* Add method to the concern service to fetch all concerns from the database when given an intake_id. The `intake` table has a field that implements many to many relationship with the `concern` table. There needs to be a many to many query done with the intake_id param on the intake table.



## Steps to test
1. Run the tests to verify the method works. There is one success test added called - `test_get_familial_concern_by_intake_id_success(concern_service)`



## What should reviewers focus on?
* How to implement the many to many to query? Resources that might help - 
( I tried them out but maybe I was wrong in my implementiation) - https://stackoverflow.com/questions/25668092/flask-sqlalchemy-many-to-many-insert-data
https://stackoverflow.com/questions/25668092/flask-sqlalchemy-many-to-many-insert-data


## Checklist
